### PR TITLE
Added constructor and this keyword Documentation

### PR DESCRIPTION
**Pull request by Aprakavdia02**
<hr>

## Description 📜
Added Documentation for the constructor and this keyword 
Fixes #2615 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [x] Java_Domain

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Java/Classes and Methods/Constructors and This keyword/constructor_and_this_keyword.ipynb [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "G7SMKVbyMqav"
+      },
+      "source": [
+        "**COPY CONSTRUCTOR:**\n",
+        "\n",
+        "Now, Java also has a copy constructor. But with a twist. Generally, the copy constructor is a constructor which creates an object by initializing it with an object of the same class, which has been created previously. Java does not provide an explicit copy constructor we need to define it ourselves.\n"
```
Comment (AkMo3) [inline Java/Classes and Methods/Constructors and This keyword/constructor_and_this_keyword.ipynb:158]: `Generally, the copy constructor is a constructor...` plag from https://www.tutorialspoint.com/what-are-copy-constructors-in-java. Please change the language

**Review by AkMo3 — CHANGES_REQUESTED** [review]

**Comment by AkMo3** [comment]
Hi @Aprakavdia02, please do a grammatical and spell check (By any online tool) and do the necessary changes and push again. I found a lot of grammatical errors so please make our work a bit easy by doing the correction yourself!

**Comment by Aprakavdia02** [comment]
@AkMo3 I have fixed the issues. Now do I have to create a new pull request or add the file in this pull request itself

**Review by AkMo3 — APPROVED** [review]
@garvitraj Do the final review

**Comment by AkMo3** [comment]
> 
> 
> @AkMo3 I have fixed the issues. Now do I have to create a new pull request or add the file in this pull request itself

No, all the changes in your branch are automatically reflected here.

**Comment by Aprakavdia02** [comment]
@garvitraj the changes u asked me to do are done.

**Review by garvitraj — APPROVED** [review]

**Review by garvitraj — APPROVED** [review]
Good work

**Pull request merged**